entropylab/results/dashboard/dashboard.py

```diff
@@ -0,0 +1,181 @@
+import json
+
+import dash
+import pandas as pd
+from dash import html, dcc
+from dash.dependencies import Input, Output, State, ALL
+import dash_bootstrap_components as dbc
+from plotly import graph_objects as go
+from plotly.subplots import make_subplots
+
+
+from entropylab import SqlAlchemyDB
+from entropylab.results.dashboard.dashboard_data import SqlalchemyDashboardDataReader
+from entropylab.results.dashboard.layout import layout
+from entropylab.results.dashboard.theme import (
+    colors,
+    plot_legend_font_color,
+    plot_paper_bgcolor,
+    plot_plot_bgcolor,
+    plot_font_color,
+)
+from entropylab.results_backend.sqlalchemy.project import project_name, project_path
+
+MAX_EXPERIMENTS_NUM = 10000
+EXPERIMENTS_PAGE_SIZE = 6
+
+_plot_figures = {}
+_plot_ids_to_combine = []
+
+
+def init(app, path):
+    """Initialize the results dashboard Dash app to display an Entropy project
+
+    :param app the Dash app to initialize with the dashboard contents
+    :param path path where the Entropy project to be used resides."""
+
+    # App setup
+
+    _title = f"Entropy - {project_name(path)} [{project_path(path)}]"
+    app.title = _title
+
+    # Fetching data from DB
+
+    dashboard_data_reader = SqlalchemyDashboardDataReader(SqlAlchemyDB(path))
+    experiments = pd.DataFrame(
+        dashboard_data_reader.get_last_experiments(MAX_EXPERIMENTS_NUM)
+    )
+    # TODO: How to filter this column when its values are emoji?
```

Comment:
maybe worth disabling the filter until it is solved?

Reply:
I'd rather keep the filter as is for this PR and work on a fix for it immediately after in a separate PR if that's alright with you?

Reply:
I'm OK with not removing this code on the current PR